### ISSUE TITLE
Multiple events

### ISSUE THREAD

**Issue by KPS-gp** [Expecho/Self-Hosted-Asp.Net-WebHooks#8]
Hi, I would like to thank you about your project. 
I'm researching about webhooks stuff, I would like to set webhooks between 2 programs (an api hosted on azure and a windows client), on internet many examples are based on cloud solution. So I suppose your project is the best to do it. About this, what will be the best words to google it? "Self hosted webhooks", "Custom webhooks"?

So I tried debugging your sample, it's works. 
I added a level wich is saving persitence in my sql server, in order to keep in memory subscriptions . I did simulation with connection lost , when client reconnect it's ok, but client receive twice same events and then disconnect / reconnect , one more. 
What do i need to change to have unique event even after disconnect / reconnect ?

Thank you.

**Comment by Expecho** [comment]
> I did simulation with connection lost , when client reconnect it's ok, but client receive twice same events and then disconnect / reconnect , one more.

This could happen if the clients registers itself again upon reconnecting. Are you sure you are not accidentally post the registration again.

**Comment by KPS-gp** [comment]
> > I did simulation with connection lost , when client reconnect it's ok, but client receive twice same events and then disconnect / reconnect , one more.
> 
> This could happen if the clients registers itself again upon reconnecting. Are you sure you are not accidentally post the registration again.

Yes, it's surely that, but how register client when disconnect or restart. Maybe I don't understood basics? So, If i 'm thinking about registration / unregistration, nothing todo when client restart, because with first registration server make the reconnection?

**Comment by Expecho** [comment]
So you modified my example to include persistance, is there a fork I can take a look at?

**Comment by KPS-gp** [comment]
Sure, my fork is pushed.
